group_vars/all

```diff
@@ -116,19 +130,23 @@ galaxy_extras_config_uwsgi: true
 galaxy_extras_config_galaxy_job_metrics: true
 galaxy_extras_config_slurm: true
 galaxy_extras_config_galaxy_root: true
-galaxy_extras_config_ie_proxy: false
 supervisor_manage_nginx: true
 supervisor_manage_proftp: true
 supervisor_manage_slurm: true
 supervisor_manage_reports: true
 supervisor_manage_docker: false
 supervisor_manage_nodeproxy: false
-supervisor_manage_ie_proxy: false
 proftpd_nat_masquerade: false
 supervisor_proftpd_autostart: true
 nginx_upload_store_path: "/home/galaxy/galaxy/database/tmp/upload_store"
 galaxy_extras_config_cvmfs: false
 
+# interactive environments installation: switch to true
```

Comment:
`To enable interactive environments set the following variables to true` ?

Reply:
@mvdbeek wrote
> The downside of this PR is that we're not automatically building the docker image anymore, and we are not testing a bunch of scenarios, like a different proxy prefix and import/export of data. Definitely the deployment script is going to fail. Maybe we can add a second travis environment that runs the docker test ?

I tried to switch to a test that more closely parallels the playbook, but I agree and I am concerned too by the lack of proxy prefix testing, or import export of data. Automated docker deployment is also something that would be stupid to loose. Thus, Yeah, a second travis environment would be ideal - I still have to figure out how to. Do you have an example in mind in a public repo ?